instrumentation/p6spy/README.md

```diff
@@ -33,4 +33,13 @@ includeParameterValues=true
 excludebinary=true
 ```
 
+`spy.properties` applies globally to any instrumented jdbc connection. If you need more flexibility to set the remoteServiceName you can add
```

Comment:
`spy.properties` applies globally to any instrumented jdbc connection. To override this, add the `zipkinServiceName` property to your connection string.